beets/autotag/mb.py

```diff
@@ -270,7 +270,7 @@ def track_info(recording, index=None, medium=None, medium_index=None,
         info.arranger = u', '.join(arranger)
 
     # Supplementary fields provided by plugins
-    extra_trackdatas = plugins.send('mb_track_extract', data=recording)
+    extra_trackdatas = plugins.send('mb_track_extract', info=recording)
```

Comment:
Minor nit: data is already plural (singular is "datum"). Maybe use "datasets"?

Reply:
done!